### ISSUE TITLE
Keybindings are destructive

### ISSUE THREAD

**Issue by 19007361** [Henoc/svgeditor#74]
Very good extension. 
I must however ask if there couldn't be other default keybindings rather than `[`, `]`, `=`, `-` and the backspace key?
These are all regularly used and seem to apply globally rather than only when the SVG Editor is open, I had to remove them.
Thank you

**Comment by itzTerra** [comment]
Yes, I couldn't use backspace in normal editors after some actions. Also, #45 seems to be related.
